.travis.yml

```diff
@@ -5,8 +5,9 @@ python:
   - '3.6'
 
 install:
-    - pip install --upgrade setuptools
-    - pip install -r requirements-dev.txt
+  - pip install --upgrade setuptools
+  - pip install -r requirements-dev.txt
 
 script:
-    flake8 demos
+  - flake8 demos
+  - make doc-spelling
```

Comment:
I think you should do `make docs` here as well to ensure that they will build correct.

Reply:
Not sure..
Spell checker fails on incorrect input too AFAIK.
+0 for change.